### DISCUSSION TITLE
Dynamic DNS (url) cache too long. how to deal with it?

### DISCUSSION THREAD

**Discussion by lessload** [DNSCrypt/dnscrypt-proxy#1817] [Q&A]
When connect to some server use dynamic dns ( like `xxxx.duckdns.org` ).
Connection was lost about 15 minute when ip of `xxxx.duckdns.org` was changed.
From my test (disable client cache) seem like problem come from dnscrypt server cache it too long, about 15 minute.
How to deal with it?
( try Forwarding rules but fail becuase i use config `force_tcp = true` )

**Comment by jedisct1** [comment]
Looks at the configuration parameters with `ttl` in their names.

**Reply by lessload** [reply]
from i test
client side `cache = false` >> server side ( i'm not owner of it, can't change their setting )
`ttl` should not effect rigth?
